build/common.props

```diff
@@ -8,10 +8,25 @@
     <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)Key.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly>
     <PublicSign Condition="'$(OS)' != 'Windows_NT'">true</PublicSign>
+    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
```

Comment:
Not a workaround. We need this because we are on pre-release versions of NETStandard.Library and Microsoft.NETCore.App. We can consider removing it once we get to 2.0.0 stable version.